### ISSUE TITLE
PHP Warning:  http_build_query(): … in rmccue/requests

### ISSUE THREAD

**Issue by dnaber-de** [wp-cli/wp-cli#3501]
Using WP-CLI (`0.24.1`) and running `wp post delete 82418 --force --defer-term-count` I see the following error:

```
PHP Warning:  http_build_query(): Parameter 1 expected to be Array or Object.  Incorrect value given in phar:///var/www/bin/wp-cli.phar/vendor/rmccue/requests/library/Requests/Transport/cURL.php on line 329
Warning: http_build_query(): Parameter 1 expected to be Array or Object.  Incorrect value given in phar:///var/www/bin/wp-cli.phar/vendor/rmccue/requests/library/Requests/Transport/cURL.php on line 329
Success: Deleted post 82418.
```

The reason is, that I have ElasticPress enabled which triggers an HTTP request. As far as I can see, it uses the WordPress function `wp_remote_request()` with [this parameters](https://github.com/10up/ElasticPress/blob/develop/classes/class-ep-api.php#L257). I don't see any errors when indexing via WP-CLI or when deleting a post via WP admin GUI.

**Comment by danielbachhuber** [comment]
Hi @dnaber-de,

Github issues are no longer an appropriate venue for support. Please review http://wp-cli.org/#support

**Comment by dnaber-de** [comment]
This isn't a support request, it's a bug report.

**Comment by danielbachhuber** [comment]
> The reason is, that I have ElasticPress enabled which triggers an HTTP request.

This isn't a WP-CLI bug report.

**Comment by dnaber-de** [comment]
Well obviously the Requests-lib shipped with WP-CLI is incompatible with that one in WP-Core. This can _only_ be handled by WP-CLI.

**Comment by danielbachhuber** [comment]
> Well obviously the Requests-lib shipped with WP-CLI is incompatible with that one in WP-Core. This can _only_ be handled by WP-CLI.

Requests was updated in WP-CLI v0.25.0 to the same version as that in WordPress core, see #3315

Please update your copy of WP-CLI to the latest.

**Comment by dnaber-de** [comment]
Thanks. I didn't noticed the available update (`0.25.0`).
